workers/event-manager/src/worker.rs

```diff
@@ -0,0 +1,67 @@
+//! Defines the worker implementation for the event manager
+
+use std::thread::JoinHandle;
+
+use async_trait::async_trait;
+use common::{types::CancelChannel, worker::Worker};
+use job_types::event_manager::EventManagerReceiver;
+use libp2p::Multiaddr;
+
+use crate::error::EventManagerError;
+
+// ----------
+// | Config |
+// ----------
+
+/// The configuration for the event manager
+pub struct EventManagerConfig {
+    /// The address to export relayer events to
+    pub event_export_addr: Option<Multiaddr>,
```

Comment:
essentially to make the instantiation of & interacting with the events manager worker simpler. this is inspired by the pattern we use in the price reporter, where we will always create the worker (and the send/receive/cancel channels, etc.), but we will ignore jobs in the worker's core execution loop if its disabled.

if we didn't do this and instead conditionally created the worker & channels based on `if event_export_addr.is_some()`, it would make sending events to the worker across the codebase much hairier